include/rdma/fi_ext.h

```diff
@@ -43,22 +43,25 @@ extern "C" {
 #endif
 
 /*
- * Each provider can define provider-specific values by
- * choosing an unique 11-bit (since the most significant
- * bit must be 1) provider-specific code to avoid
- * overlapping with other providers. E.g.,
+ * Each provider needs to define an unique 12-bit provider
```

Comment:
The higher 4 bits are left to 0 intentionally?

Reply:
Yes, we do not touch the highest 4 bits when defining the macro `FI_PROV_SPECIFIC_XXX`, so those 4 bits are left to 0.

Reply:
I see. Can you add that to the comments?

Reply:
Updated in new revision.